test/images/node-perf/npb-ep/BASEIMAGE

```diff
@@ -0,0 +1 @@
+amd64=debian:stretch
```

Comment:
only amd64? cc @dims @mkumatag

Reply:
try the following?

```
amd64=debian:stretch
arm=arm32v7/debian:stretch
arm64=arm64v8/debian:stretch
ppc64le=ppc64le/debian:stretch
```

Reply:
also our debian-base images are smaller, but that might not matter much for a test image.

Reply:
I'm having issues building the workloads for some archs. For `npb-ep` and `npb-is`, builds work for all architectures other than arm32. For tf-wide-deep, I'm not able to build the images for anything other than amd64. Do you know if Tensorflow supports archs other than amd64?